views/BuzzerView/BuzzerView.tsx

```diff
@@ -24,7 +24,12 @@ export const BuzzerView: FC<BuzzerViewProps> = ({ route }) => {
 
   return (
     <LayoutProvider
-      navbar={<Navbar text={`${location} - ${nodeId} - ${buzzer?.name}`} />}>
+      navbar={
+        <Navbar
+          text={`${location} - ${nodeId} - ${buzzer?.name}`}
+          variant="header-small"
```

Comment:
I'm not fond of inconsistent navbar font size, it should be the same in all views. Please apply it everywhere as `header-small`